src/ui/public/chrome/directives/app_switcher_link/app_switcher_link.less

```diff
@@ -22,18 +23,19 @@
       text-decoration: none;
     }
   }
+
+  .is-app-links-wrapper-open & {
+    width: @as-open-width;
+  }
```

Comment:
Actually, after staring at it for a little longer, it's not that bad. I'll make the change.

The things I do for Bargs...